src/Command.js

```diff
@@ -83,8 +96,8 @@ export default class Command {
   }
 
   runValidations() {
-    if (this.concurrency < 1) {
```

Comment:
Turns out that this block could _never_ be entered because of the way we currently interpret the `sort` option.